backend-ktor/src/main/kotlin/me/zwsmith/plugins/Models.kt

```diff
@@ -0,0 +1,22 @@
+package me.zwsmith.plugins
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Recipe(
+    val id: Int,
+    val name: String,
+    val description: String,
```

Comment:
This feels like it should be nullable/optional, otherwise you'll have to store blank strings as descriptions and that would lead to annoying validation